pkg/controllers/repositoryserver/utils.go

```diff
@@ -59,6 +59,21 @@ const (
 	tlsCertDefaultMountPath = "/mnt/secrets/tlscert"
 	tlsKeyPath              = "/mnt/secrets/tlscert/tls.key"
 	tlsCertPath             = "/mnt/secrets/tlscert/tls.crt"
+
+	serverSetupErrReason     string = "KopiaRepositoryServerSetupFailed"
+	serverSetupSuccessReason string = "KopiaRepositoryServerSetupSucceeded"
+
+	repositoryConnectedErrReason     string = "KopiaRepositoryConnectionFailed"
+	repositoryConnectedSuccessReason string = "KopiaRepositoryConnectionSucceeded"
+
+	serverInitializedErrReason     string = "KopiaRepositoryServerInitializationFailed"
+	serverInitializedSuccessReason string = "KopiaRepositoryServerInitializationSucceeded"
+
+	clientsInitializedErrReason     string = "ClientInitializationFailed"
+	clientsInitializedSuccessReason string = "ClientInitializationSuccessful"
```

Comment:
[nit]

```suggestion
	clientsInitializedSuccessReason string = "ClientInitializationSucceded"
```

All other reason fields have succeded appended to it , hence lets be consistent with it